maas/plugins/hp_monitoring.py

```diff
@@ -59,7 +59,7 @@ def get_controller_status():
 
 def get_controller_cache_status():
     return check_command(('hpssacli', 'ctrl', 'all', 'show', 'status'),
-                         'Cache Status', 'OK')
+                         'Cache Status', ('OK', 'Not Configured'))
```

Comment:
Yeah, I was testing on my DL380 Gen8 and got this (I think you are using Gen9s):

root@rpc-openstack:~# hpssacli ctrl all show status

Smart Array P420i in Slot 0 (Embedded)
   Controller Status: OK
   Cache Status: Not Configured
   Battery/Capacitor Status: OK

Usually you don't configure cache if you're using SSDs as it actually slows down the array.  I can remove though if you think it'll be an issue.